### ISSUE TITLE
Druid Performance After 0.31.2

### ISSUE THREAD

**Issue by Jhors2** [metabase/metabase#10143]
We've noticed significant query performance degradation for new "question" building after upgrading to the 0.32 line.  

Queries with no aggregations (typically in the case of data discovery use cases) generate select queries against our Druid cluster.  These queries are incredibly slow and sometimes fail to complete entirely.  It would be very useful to investigate the Scan query type or return to the old method for querying druid without aggregations.

**Comment by salsakran** [comment]
Calling this a P1 since we have lots of natural clicks in the app which lead to a table listing, and it that's especially non-performant we're being really bad citizens.

**Comment by camsaul** [comment]
@sbelak would you know off the top of your head what recent changes in our Druid might be causing a performance regression?

**Comment by sbelak** [comment]
No, and I don't see any Druid-specific change that would cause it to issue scan queries.

**Comment by camsaul** [comment]
@sbelak I think what @Jhors2 is saying is that we should look into using Scan queries instead of Select.

@Jhors2 as far as I know we've always issued Select queries if there are no aggregations. I don't think Scan queries existed at the time I wrote the Druid driver. Select queries were new at the time and considered an experimental feature, but I used them because they offered more powerful filtering than some of the other query types.

**Comment by flamber** [comment]
Closing, since 0.35 changed Select to Scan. Please comment if you're still seeing performance issues.
PR #11992
